bundles/org.openhab.binding.remoteopenhab/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.nest-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
This suggestion will be automated by https://github.com/openhab/static-code-analysis/pull/395:

```suggestion
<features name="org.openhab.binding.remoteopenhab-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Reply:
Done.
I run "mvn spotless:apply" too.